sources/engine/Stride.Rendering/Rendering/Shadows/ShadowMapRenderer.cs

```diff
@@ -44,9 +42,6 @@ public ShadowMapRenderer()
 
         public HashSet<RenderView> RenderViewsWithShadows { get; } = new HashSet<RenderView>();
 
-        // TODO
-        public IReadOnlyList<RenderStage> ShadowMapRenderStages => shadowMapRenderStages;
```

Comment:
Clearly we don't know what was the idea behind this. Since this is part of public API, can you mark your PR as 'breaking changes' for awareness?

Reply:
Will do.